pkg/webhook/validate.go

```diff
@@ -22,7 +22,7 @@ const (
 
 // SupportedModels holds the NIC models officially supported
 var SupportedModels = map[string]([]string){
-	IntelID:    []string{"158b"},
+	IntelID:    []string{"158b", "0d58"},
```

Comment:
This probably belongs to the second part?

Reply:
I remove it now.

not related to any PR just wanted to add another network card as supported one (the one I have in my server) to test both the accelerators and the network functionality